packages/react-core/elements/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aws-amplify/ui-react-core/elements",
   "main": "../dist/elements.js",
-  "module": "../dist/esm/elements/elements.mjs",
+  "module": "../dist/esm/elements.mjs",
```

Comment:
Required due to changes in the bundled output ([ref](https://github.com/aws-amplify/amplify-ui/commit/1f5099b7ba57b520052e240dc4790b3f94bc182b#diff-90e3b6e94cbbc7af3ac13252c2e21a4f32f70ed8de844312a42b0d61644bb3d0R1))